google/cloud/forseti/services/server_config/service.py

```diff
@@ -26,6 +29,54 @@
 LOGGER = logger.get_logger(__name__)
 
 
+def _run(client):
+    """Runs Forseti.
+
+    Args:
+        client (ClientComposition): Forseti client object.
+
+    Returns:
+        ServerRunReply: the returned proto message.
+    """
+
+    inventory_id = 0
+
+    for progress in client.inventory.create():
+        inventory_id = progress.id
+
+    inventory_meta = client.inventory.get(inventory_id)
+
+    if inventory_meta.errors:
+        message = 'Error create inventory: {}'.format(inventory_meta.errors)
+        return server_pb2.ServerRunReply(message=message)
+
+    today = datetime.now()
+    model_name = today.strftime('run_model_%Y%m%d_%H%M')
+    model_reply = client.model.new_model(
+        'inventory',
+        model_name,
+        inventory_id,
+        False)
+
+    model_handle = model_reply.model.handle
+    model_status = model_reply.model.status
+
+    client.switch_model(model_handle)
+
+    if model_status not in ['SUCCESS', 'PARTIAL_SUCCESS']:
+        message = 'ERROR: Model status is {}'.format(model_status)
```

Comment:
I think that you need to add a step to delete model here.  And also a return the message step.